.github/workflows/aws-deploy.yaml

```diff
@@ -10,6 +10,9 @@ on:
       role-to-assume:
         required: true
         type: string
+      role-session-name:
```

Comment:
this PR is not complete.  you need to actually pass the role session name in from the deploy.yaml template.

Reply:
@zaro0508 I don't understand this comment.  Isn't the change to `deploy.yaml`, below, the required change?

Reply:
yes, you are correct.  i somehow missed it.  sorry.